pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
 python = ">=3.7.0,<4"
 datadog = ">=0.41.0,<1.0.0"
 wrapt = "^1.11.2"
-ddtrace = "^1.6.4"
+ddtrace = "^1.14.0"
```

Comment:
my main concern is that we bumped many versions, I'm not sure how we handle this.